src/components/Waveform.js

```diff
@@ -1,9 +1,89 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Peaks from 'peaks.js';
+import { Button, ButtonToolbar, FormControl, Form } from 'react-bootstrap';
 
-const Waveform = () => (
-  <section className="waveform-section">
-    <div className="waveform-holder" />
-  </section>
-);
+import soundMP3 from '../data/TOL_6min_720p_download.mp3';
+import soundOGG from '../data/TOL_6min_720p_download.ogg';
+import soundJSON from '../data/TOL_6min_720p_download.json';
+import soundDAT from '../data/TOL_6min_720p_download.dat';
+
+class Waveform extends Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			peaksInstance: null,
+			seekTime: ''
+		};
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	componentDidMount() {
+		const options = {
+		  container: document.getElementById('waveform-visualiser-container'),
+		  mediaElement: document.querySelector('audio'),
+		  dataUri: {
+            arraybuffer: soundDAT,
+            json: soundJSON
+          },
+		  keyboard: true,
+		  pointMarkerColor: '#006eb0',
+		  showPlayheadTime: true
+		};
+
+		this.setState({
+			peaksInstance: Peaks.init(options)
+		});
+	}
```

Comment:
Generally, in React using `document.getElementById()` and other vanilla JS targeting is an anti-pattern.  To get this through, and if it works that's fine, however check out React's `ref`.  It's the "React way" of targeting an element when you need a direct reference to a DOM element.